src/components/Tooltip.tsx

```diff
@@ -36,7 +36,8 @@ export function Tooltip(props: TooltipProps) {
         {...(!state.isOpen && typeof title === "string" ? { title } : {})}
         {...tid}
         // Add display contents to prevent the tooltip wrapping element from short-circuiting inherited styles (i.e. flex item positioning)
-        css={Css.display("contents").$}
+        // Once the element is `:active`, allow pointer events (i.e. click events) to pass through to the children.
+        css={Css.display("contents").addIn(":active", Css.add("pointerEvents", "none").$).$}
```

Comment:
I tried to write a test for this, but couldn't figure out how to get a click event to bubble up in tests. After 5-10minutes of searching and tryings out, I figured a manual test was good enough for now given the state of Beam soon being sunsetted.